app/models/export.rb

```diff
@@ -1,6 +1,6 @@
 require "csv"
 
-class Export < ActiveRecord::Base
+class Export # Not ActiveRecord Base
```

Comment:
I think we can take out this comment.  It's not uncommon to have models that aren't active record models.